example/src/App.re

```diff
@@ -1,10 +1,12 @@
-let component = ReasonReact.statelessComponent("Home");
+open BsReactNavigation;
 
-let reactClass =
-  BsReactNavigation.Navigation.createStackNavigator(
-    Home.reactClass,
-    Detail.reactClass,
-  );
+let navigationConfig =
+  StackNavigator.createNavigationConfig(~initialRouteName="Home");
+
+let routes = [
+  ("Home", StackNavigator.createRoute(~screen=() => <Home />)),
```

Comment:
The cool thing is you we can use [@bs.deriving jsConverter] on polymorphic variant and have the route type which can be used in `navigate`

Reply:
I was playing with functors to solve a problem with the nested navigator. 
Still, need to resolve some problems but going forward. Will be public that concept soon :)